app/Http/Controllers/V2/Dashboard/VolunteersAndAverageSurvivalRateController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Http\Controllers\V2\Dashboard;
+
+use App\Helpers\TerrafundDashboardQueryHelper;
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\Dashboard\VolunteersAndAverageResource;
+use Illuminate\Http\Request;
+
+class VolunteersAndAverageSurvivalRateController extends Controller
+{
+    public function __invoke(Request $request): VolunteersAndAverageResource
+    {
+        $projects = TerrafundDashboardQueryHelper::buildQueryFromRequest($request)->get();
+
+        $response = (object)[
+            'total_volunteers' => $this->getTotalVolunteerSum($projects),
+            'men_volunteers' => $this->getVolunteersSum($projects, 'volunteer_men'),
+            'women_volunteers' => $this->getVolunteersSum($projects, 'volunteer_women'),
+            'youth_volunteers' => $this->getVolunteersSum($projects, 'volunteer_youth'),
+            'non_youth_volunteers' => $this->getVolunteersSum($projects, 'volunteer_non_youth'),
+            'non_profit_survival_rate' => $this->getAverageSurvivalRate($projects, 'non-profit-organization'),
+            'enterprise_survival_rate' => $this->getAverageSurvivalRate($projects, 'for-profit-organization'),
+            'number_of_sites' => $this->numberOfSites($projects),
+            'number_of_nurseries' => $this->numberOfNurseries($projects),
+        ];
+
+        return new VolunteersAndAverageResource($response);
```

Comment:
Since this resource is doing nothing other than mapping what you have above again, this would be better done with (and remove the resource class)
```
return response()->json($response);
```